swift/onnxTests/SwiftOnnxWrapperTests.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import XCTest
+import Foundation
+@testable import OnnxWrapper
+
+final class SwiftOnnxWrapperTests: XCTestCase {
+    func testExample() throws {
+        let modelPath: String = ""
+        let threadCount: Int32 = 1
+
+        do {
+            let env = try ORTEnv(loggingLevel: ORTLoggingLevel.verbose)
+            let options = try ORTSessionOptions()
+            try options.setLogSeverityLevel(ORTLoggingLevel.verbose)
+            try options.setIntraOpNumThreads(threadCount)
```

Comment:
Could we also test adding the CoreML and XNNPACK EPs (they should be included in the default iOS ORT package? 

Example code to register each is in the tests. Not sure how to translate to Swift though. 

https://github.com/microsoft/onnxruntime/blob/main/objectivec/test/ort_session_test.mm#L196-L240

Reply:
do you want me to add GitHub action to run this tests?

Reply:
We typically run that sort of test from the CI configs under \tools\ci_build\github\azure-pipelines. 

Could the build/test be incorporated into or be setup in a similar way to \tools\ci_build\github\apple\test_ios_packages.py, which is called by \tools\ci_build\github\apple\build_and_assemble_ios_pods.py?

build_and_assemble_ios_pods.py called from \tools\ci_build\github\azure-pipelines\mac-ios-packaging-pipeline.yml in the packaging pipeline.

Reply:
Not sure. Our pipeline looks like this:
https://onedrive.visualstudio.com/DefaultCollection/Clipchamp/_git/clipchamp-ios-composition-kit.git?path=/.azure/pipelines/unit-tests.yml

Reply:
Added tests you've asked for